### ISSUE TITLE
Potential solutions for #3820 Performance and file size issue (smart clips) 

### ISSUE THREAD

**Issue by virgilON** [audacity/audacity#3963]
### Your idea

I am one of those "poor old users" that was broadsided by this bug.  Some of my files grew to 35GB(!), so I've had to come up with a salvage routine (finding no suggestions in the forums).  My projects involve numerous Label Tracks in addition to Audio.  I export the audio from a bloated project to WAV files, then open these in a new project and Copy/Paste them back into the original as new tracks.  After aligning the new tracks (they seem to have a ~13ms buffer tacked on the front that I delete) I can transfer (by snapping to) any splits from the original track(s) that I want to preserve.  The original audio tracks are then deleted and the project saved.  The project size reverts to something reasonable, but only after the project is closed.  It's a PITA workaround, but doable.

I suggest two possible solutions to the bug:  1) a 'five second rule': limit the data that a smart clip grabs to five seconds on either side of the clip (the value could be changed from 0 - whatever in Preferences perhaps);  and/or 2) a 'purge' function that would permanently remove the hidden data (essentially automating my workaround).  I don't really like this second approach, but would advocate including it in the next version or two in order to allow users to clean up the bloated messes that the current version is creating. 

### Problem to be solved

#3820 Performance and file size issues when copying and pasting a small portion of a very large clip (into a different project)

### Additional context

@LWinterberg, @Tantacrul
This bug is an urgent problem that should be addressed in the very next release, please!
I recognize that this is not a new issue; however, the original thread is locked and not showing recent activity.

**Comment by LWinterberg** [comment]
Duplicate of https://github.com/audacity/audacity/issues/3820 - the issue in question has a fix awaiting review
